src/Sheet/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import SheetContainer from './SheetContainer';
+
+export const POSITIONS = {
+  left: 'left',
+  right: 'right',
+  top: 'top',
+  bottom: 'bottom',
+};
+
+export const VARIANTS = {
+  light: 'light',
+  dark: 'dark',
+};
+
+class Sheet extends React.Component {
+  constructor(props) {
+    super(props);
+    this.blockClicks = this.blockClicks.bind(this);
+  }
+
+  blockClicks(event) {
+    event.stopPropagation();
+  }
+
+  render() {
+    const {
+      blocking,
+      children,
+      position,
+      show,
+      variant,
+    } = this.props;
+    if (!show) {
+      return null;
+    }
+    return (
+      <SheetContainer>
+        <div
+          className={classNames(
+            'pgn__sheet-skrim',
+            { hidden: !blocking },
+          )}
+          onClick={this.blockClicks}
+          role="presentation"
+        />
+        <div
+          className={classNames(
+            'pgn__sheet-component',
+            `pgn__sheet__${variant}`,
+            position,
+
+          )}
+          role="alert"
```

Comment:
@wittjeff I know there was some discussion on the sheet component using the alert role, but I'm guess this role would be odd if the sheet were used for a side nav or some use case like it. Is this something we should bake in, make configurable, or leave to child elements to define?